materials/internals/definition.yaml

```diff
@@ -171,6 +171,10 @@ states:
 componentGroups:
   input:
     description: 'Input elements'
+  tag:
+    description: 'Tag elements'
+  tag-warning:
```

Comment:
wasn't 100% sure on whether to allow `multiple states` or to put it as a separate component group;